docs/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -146,3 +146,7 @@ const httpLink = createHttpLink({
 With this configuration, Replay is able to capture response bodies from Apollo Client requests.
 
 </Expandable>
+
+<Expandable title="`console` calls are triggering replays to record">
+Using the [`Capture Console Integration`](https://docs.sentry.io/platforms/javascript/configuration/integrations/captureconsole/) can cause replays to record as if you have triggered an exception. You can use the [`beforeErrorSampling`](https://docs.sentry.io/platforms/javascript/guides/sveltekit/session-replay/understanding-sessions/#ignore-certain-errors-for-error-sampling) to avoid this behavior.
```

Comment:
```suggestion
Using the [`captureConsoleIntegration`](https://docs.sentry.io/platforms/javascript/configuration/integrations/captureconsole/) can cause replays to record as if you have triggered an exception. You can use [`beforeErrorSampling`](https://docs.sentry.io/platforms/javascript/guides/sveltekit/session-replay/understanding-sessions/#ignore-certain-errors-for-error-sampling) to avoid this behavior.
```